### DISCUSSION TITLE
WebSocket support?

### DISCUSSION THREAD

**Discussion by MarkusBiggus** [Windos/BurntToast#86] [General]
This is a fantastic package, thanks for doing it.
Is there any chance the HTTP Listener could have a variation where it converts the connections to websocket? It's not particularly difficult, just my powershell skill is at the level I can read it OK but not really write anything advanced.
I want to use clients with a connection to a message router via websockets and they received the toast packet from there. 
It'll be used by an app to send messages with a url to users who are not using the app with a link to the page they need to open. Imagine a workflow scenario where the workflow is handed off to someone who may not be in the app itself but logged and reachable.

**Comment by Windos** [comment]
So sorry for not replying sooner, @MarkusBiggus.

The HTTP listener you're referring to isn't a feature of this project, but an example use case submitted during [Hacktoberfest](https://hacktoberfest.digitalocean.com/). I imagine using websockets are also a perfectly valid use case, but it's not something I've personally played with.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
